lib/DaxMailer/Schema/Result/Bang.pm

```diff
@@ -12,10 +12,12 @@ primary_column url     => { data_type => 'text' };
 
 column email_address   => { data_type => 'text', is_nullable => 1 };
 column comments        => { data_type => 'text', is_nullable => 1 };
+column example_search  => { data_type => 'text', is_nullable => 1, default_value => 'hello' };
 column site_name       => { data_type => 'text' };
 column category_id     => { data_type => 'integer' };
 
 column moderated       => { data_type => 'integer', default_value => 0 };
+column created         => { data_type => 'timestamptz', set_on_create => 1, default_value => '2019-01-01' };
```

Comment:
If I'm understanding this correctly...

- `set_on_create` is a DBIx thing that sets the timestamp correctly when an entry is created
- `set_on_create` will trump `default_value`, so all entries created before this is released will be marked as created on `2019-01-01`, and all newly inserted entries will have the correct time

Does that sound about right?

Reply:
Yeah, as I understand it `set_on_create` is part of the DBIC TimeStamp extension which sets a default value in Perl. The `default_value` is an instruction to the deployment code and is reflected in the DDL alone.

We can write a test to assert this.